win32ss/gdi/ntgdi/freetype.c

```diff
@@ -3562,8 +3563,8 @@ IntRequestFontSize(PDC dc, PFONTGDI FontGDI, LONG lfWidth, LONG lfHeight)
     EmHeight = max(EmHeight, 1);
     EmHeight = min(EmHeight, USHORT_MAX);
 
-    if (pOS2 && lfWidth > 0)
-        Width64 = FT_MulDiv(lfWidth, face->units_per_EM, pOS2->xAvgCharWidth) << 6;
+    if (lfWidth != 0)
+        Width64 = (FT_MulDiv(lfWidth, 96 * 5, 72 * 3) << 6); /* ??? FIXME */
```

Comment:
This might cause your failing unittest?

Reply:
Visualization says the latter is better. But tests say the former is better.